src/App.js

```diff
@@ -1,28 +1,10 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
```

Comment:
- [ ] Great job working on this milestone! 💯 However, please consider removing all commented pieces of code, they make your code look dirty and less readable! 🤓 🔧

Reply:
done